src/main/java/org/springframework/data/tarantool/core/TarantoolSpaceNameResolver.java

```diff
@@ -7,7 +7,7 @@
  */
 public interface TarantoolSpaceNameResolver {
     /**
-     * Taske a class and determine the space name for it
+     * Taste a class and determine the space name for it
```

Comment:
It must be "take"

Reply:
It turned out that I corrected the typo for another typo :-) Fixed it.